src/maxdiffusion/tests/train_smoke_test.py

```diff
@@ -43,24 +43,27 @@ def setUp(self):
 
   def test_sdxl_config(self):
     output_dir="train-smoke-test"
+    run_name="sdxl_train_smoke_test"
     train_sdxl_main([None,os.path.join(THIS_DIR,'..','configs','base_xl.yml'),
       "pretrained_model_name_or_path=stabilityai/stable-diffusion-xl-base-1.0",
-      "revision=refs/pr/95","dtype=bfloat16","run_name=sdxl_train_smoke_test",
+      "revision=refs/pr/95","dtype=bfloat16",f"run_name={run_name}",
       "max_train_steps=21","dataset_name=diffusers/pokemon-gpt4-captions",
       "resolution=1024","per_device_batch_size=1","snr_gamma=5.0",
       'timestep_bias={"strategy" : "later", "multiplier" : 2.0, "portion" : 0.25}',
-      "base_output_directory=gs://maxdiffusion-tests", f"output_dir={output_dir}"])
+      "base_output_directory=gs://maxdiffusion-tests", f"output_dir={output_dir}",
+      "cache_dir=gs://jfacevedo-maxdiffusion/cache_dir"])
```

Comment:
Should this be a configurable bucket?

Reply:
no in unit tests it is not.

Reply:
discussed offline, will be addressed in the next PR